### PR TITLE
[TACHYON-882] Adding unit tests for tachyon.util.io.PathUtils.

### PR DESCRIPTION
**Pull request by jsimsa**
Some utility functions in tachyon.util.io.PathUtils are missing unit tests; this PR adds more tests (increasing line coverage from 70% to 90%).

### CODE REVIEW

**Comment by jsimsa** [comment]
Jenkins, test this please.

File: common/src/test/java/tachyon/util/io/PathUtilsTest.java [hunk 1]
```diff
@@ -89,4 +91,98 @@ public void concatPathTest() {
...
+    // test dots gets resolved
+    Assert.assertEquals("/foo/bar", PathUtils.cleanPath("/foo/./bar"));
+    Assert.assertEquals("/foo/bar", PathUtils.cleanPath("/foo/././bar"));
+    Assert.assertEquals("/foo", PathUtils.cleanPath("/foo/bar/.."));
+    Assert.assertEquals("/bar", PathUtils.cleanPath("/foo/../bar"));
+    Assert.assertEquals("/", PathUtils.cleanPath("/foo/bar/../.."));
+
+    // the following seems strange
+    // TODO instead of returning "null", throw InvalidPathException
+    Assert.assertEquals(null, PathUtils.cleanPath("/foo/bar/../../.."));
```
Comment (apc999) [inline common/src/test/java/tachyon/util/io/PathUtilsTest.java]: NIT: better to use `assertNull` to be more specific

File: common/src/test/java/tachyon/util/io/PathUtilsTest.java [hunk 2]
```diff
@@ -89,4 +91,98 @@ public void concatPathTest() {
...
+    // test trailing slash
+    Assert.assertEquals("/foo/bar", PathUtils.cleanPath("/foo/bar/"));
+
+    // test dots gets resolved
+    Assert.assertEquals("/foo/bar", PathUtils.cleanPath("/foo/./bar"));
+    Assert.assertEquals("/foo/bar", PathUtils.cleanPath("/foo/././bar"));
+    Assert.assertEquals("/foo", PathUtils.cleanPath("/foo/bar/.."));
+    Assert.assertEquals("/bar", PathUtils.cleanPath("/foo/../bar"));
+    Assert.assertEquals("/", PathUtils.cleanPath("/foo/bar/../.."));
+
```
Comment (apc999) [inline common/src/test/java/tachyon/util/io/PathUtilsTest.java]: how do redudent slashs get resolved? e.g., "/foo//bar"?

Comment (jsimsa) [inline common/src/test/java/tachyon/util/io/PathUtilsTest.java]: Done.

File: common/src/test/java/tachyon/util/io/PathUtilsTest.java (hunk 1 above)
Comment (jsimsa) [inline common/src/test/java/tachyon/util/io/PathUtilsTest.java]: Done.

**Comment by apc999** [comment]
+1 LGTM. Thanks for the patch

**Comment by jsimsa** [comment]
When you get a chance, could you please merge this? Thanks.

**Comment by gpang** [comment]
LGTM

**Pull request merged**